src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsalertsapi/service/event/DomainEventsListener.kt

```diff
@@ -0,0 +1,23 @@
+package uk.gov.justice.digital.hmpps.hmppsalertsapi.service.event
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.awspring.cloud.sqs.annotation.SqsListener
+import org.springframework.stereotype.Service
+
+@Service
+class DomainEventsListener(
+  private val objectMapper: ObjectMapper,
+  private val personUpdatedHandler: PersonUpdatedHandler,
+) {
+  @SqsListener("hmppsdomaineventsqueue", factory = "hmppsQueueContainerFactoryProxy")
+  fun receive(notification: Notification) {
+    when (notification.eventType) {
+      PRISONER_UPDATED -> personUpdatedHandler.handle(objectMapper.readValue(notification.message))
```

Comment:
I imagine we'll also need to listen for merge events and delete the merge from prison number record so we don't accidentally add an alert for a removed number